### PR TITLE
fix: clean typography and translation

### PR DESCRIPTION
**Pull request by kecrily**
#### Prerequisites checklist

- [x] I have read the [contributing guidelines](https://github.com/eslint/eslint/blob/HEAD/CONTRIBUTING.md).

#### What changes did you make? (Give an overview)

clean typography and translation

### CODE REVIEW

**Review by harish-sethuraman — APPROVED** [review]
LGTM

File: src/rules/func-name-matching.md [hunk 1]
```diff
@@ -145,7 +145,7 @@ module['exports'] = function foo(name) {};
 
 ## 选项
 
-这个规则需要一个 "always" 或 "never" 的可选字符串（当省略时，默认为 "always"），以及一个带有两个属性 `considerPropertyDescriptor` 和 `includeCommonJSModuleExports` 的可选选项对象。
+这个规则需要一个 `always` 或 `never` 的可选字符串（当省略时，默认为 `always`），以及一个带有两个属性 `considerPropertyDescriptor` 和 `includeCommonJSModuleExports` 的可选选项对象。
```
Comment (mdjermanovic) [inline src/rules/func-name-matching.md:148]:
```suggestion
这个规则需要一个 `"always"` 或 `"never"` 的可选字符串（当省略时，默认为 `"always"`），以及一个带有两个属性 `considerPropertyDescriptor` 和 `includeCommonJSModuleExports` 的可选选项对象。
```

As these are strings.

Comment (mdjermanovic) [reply src/rules/func-name-matching.md:148]: Oops, it's already merged :) nevermind

Comment (harish-sethuraman) [reply src/rules/func-name-matching.md:148]: Oops sorry :( I can send another PR probably for this

**Pull request merged**